embed-code-go/embedding/parsing/parsing_context.go

```diff
@@ -168,12 +171,13 @@ func (pc *ParsingContext) ResolveUnacceptedEmbedding() {
 //
 // Also sets FileContainsEmbedding flag.
 func (pc *ParsingContext) SetEmbedding(embedding *embedding_instruction.EmbeddingInstruction) {
+	indexIncrease := 2 // +2 for instruction and code fence.
```

Comment:
This explanation is quite unclear. I'm not sure if it's possible to address this in the context of this PR. However, this `2` definitely needs better documentation.

It appears that this number is related to skipping content in the Markdown file and starting the insertion directly after a code marker. For example:
```
<embed-code file="path/to/file" fragment="fragment-name"></embed-code>
` ` `csharp
```

If my understanding is correct, this approach should be refactored because using `2` seems like a temporary workaround and might not work with other formats, such as:
```
<embed-code file="path/to/file" fragment="fragment-name">
</embed-code>
` ` `csharp
```

Regardless, a calculation such difficult to understand code should be encapsulated in a separate private function with clear documentation. 

It would be ideal to address this issue either now or in a future update.

Reply:
Added a `TODO` comment with an issue for that https://github.com/SpineEventEngine/embed-code/issues/48